direct/ingest-client/src/main/java/cz/o2/proxima/client/IngestClient.java

```diff
@@ -244,6 +248,101 @@ public void send(Rpc.Ingest ingest, long timeout,
     sendTry(ingest, timeout, unit, statusConsumer, false);
   }
 
+  /**
+   * Send ingest request.
+   * @param key entity key value.
+   * @param entity entity name.
+   * @param attribute attribute name.
+   * @param value ingested value.
+   * @param statusConsumer callback for receiving status.
+   */
+  public void ingest(String key, String entity, String attribute,
+                     ByteString value, Consumer<Rpc.Status> statusConsumer) {
```

Comment:
Hele also, the default intendation should be probably 6.

Reply:
yes, also changed everywhere.